turbopack/crates/turbopack-core/src/compile_time_info.rs

```diff
@@ -200,6 +200,15 @@ impl CompileTimeDefines {
     }
 }
 
+#[turbo_tasks::value]
+#[derive(Debug, Clone, Copy)]
```

Comment:
We never create a `Vc<InputRelativeConstant>`, so this should just derive whatever is necessary:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize, TraceRawVcs, NonLocalValue)]
```